src/twister2/report/subtest_reporting_plugin.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import pytest
+from pytest_subtests import SubTestReport
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_report_teststatus(report, config):
+    """ This plugin modifies the one from SubTestReport.
```

Comment:
I would put this docstring on module level as whole module is the pytest plugin.

Reply:
done